modules/material/themes/material/mfa/out-of-backup-codes.twig

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="{{ currentLanguage }}">
+<head>
+  <title>{{ '{mfa:title}'|trans }}</title>
+
+  {% include 'header.twig' %}
+</head>
+<body class="gradient-bg">
+<div class="mdl-layout mdl-layout--fixed-header fill-viewport">
+  <header class="mdl-layout__header">
+    <div class="mdl-layout__header-row">
+      <span class="mdl-layout-title">
+        {{ '{mfa:header}'|trans }}
+      </span>
+    </div>
+  </header>
+  <main class="mdl-layout__content" layout-children="column">
+    <form layout-children="column" method="post">
+      <div class="mdl-card mdl-shadow--8dp">
+        <div class="mdl-card__media white-bg margin" layout-children="column">
+          <i class="mdl-color-text--red error material-icons mdl-typography--display-4">error</i>
+        </div>
+
+        <div class="mdl-card__title center">
+          <h1 class="mdl-card__title-text">
+            {{ '{mfa:no_more_codes_header}'|trans }}
+          </h1>
+        </div>
+
+        <div class="mdl-card__title center">
+          <p class="mdl-card__subtitle-text">
+            {% if hasOtherMfaOptions %}
+              {{ '{mfa:has_options_besides_codes}'|trans }}
+            {% else %}
+              {{ '{mfa:has_no_more_options}'|trans }}
+            {% endif %}
+          </p>
+        </div>
+
+        <div class="mdl-card__actions" layout-children="row">
+          {% if hasOtherMfaOptions %}
+            <button name="continue" class="mdl-button">
+              {{ '{mfa:button_later}'|trans }}
+            </button>
+          {% endif %}
+
+          <span flex></span>
```

Comment:
how do these attributes like `flex` and `layout-children="row"` even work? Part of the MDL theme? I see its just part of the old file, but wondered if you know.

Reply:
I don't know. Sorry. Would you like to plan to do a technology/standards update for the UI code in this repo?

Reply:
Possibly. Since those aren't standard attributes I was just wondering where to find documentation on them. Not necessarily anything wrong with them.